src/web/tools/PointerEventManager.ts

```diff
@@ -190,7 +206,7 @@ export default class PointerEventManager extends EventManager<HTMLElement> {
       pointerId: event.pointerId,
       eventType: eventType,
       pointerType: event.pointerType as PointerType,
-      buttons: event.buttons,
+      button: this.mouseButtonsMapper.get(event.button),
```

Comment:
Do we care if there's more than one button pressed at the same time?

If we do, I'd rename it to `buttons`. Not sure about the format though - I'm not sure if passing a union back translates to a good ux. Maybe an array in that case?

Reply:
I don't think we should care about that since it doesn't seem to be well supported (or should I say, it doesn't seem to be supported at all).

If you click on something and then add another click with another button (for example, first you left click and then add right click to it), you won't get second event. Moreover, even if you click with two buttons at once, event seems to contain information only about "first" pointer.

So even if we consider using `event.buttons`, which contains combination of buttons, it would work only inside `pointermove`.

cc @tomekzaw

Reply:
> If you click on something and then add another click with another button (for example, first you left click and then add right click to it), you won't get second event.

It works if you subscribe to `mousedown` event, but I guess that's another thing we may want to talk about 😅